api/numaresourcesoperator/v1/numaresourcesoperator_types.go

```diff
@@ -45,6 +45,12 @@ type NUMAResourcesOperatorSpec struct {
 	// +optional
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Optional ignore pod namespace/name glob patterns"
 	PodExcludes []NamespacedName `json:"podExcludes,omitempty"`
+	// Valid values are: "Normal", "Debug", "Trace", "TraceAll".
+	// Defaults to "Normal".
+	// +optional
+	// +kubebuilder:default=Normal
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Operator log verbosity"
+	OperatorLogLevel *operatorv1.LogLevel `json:"operatorLogLevel,omitempty"`
```

Comment:
should this be an int? probably so, otherwise the mapping with the int flag is poor.